source/extensions/filters/http/csrf/csrf_filter.cc

```diff
@@ -0,0 +1,122 @@
+#include "extensions/filters/http/csrf/csrf_filter.h"
+
+#include "envoy/stats/scope.h"
+
+#include "common/common/empty_string.h"
+#include "common/http/header_map_impl.h"
+#include "common/http/headers.h"
+#include "common/http/utility.h"
+
+#include "extensions/filters/http/well_known_names.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Csrf {
+
+namespace {
+bool modifyMethod(const Http::HeaderMap& headers) {
```

Comment:
Would it make sense to name this isModifyMethod() ? Or add comment? I had to squint to figure out that the method name does mean it actually modifies anything.